kubeflow/core/iap.libsonnet

```diff
@@ -499,6 +499,7 @@
       spec: {
         rules: [
```

Comment:
What's the effect of ingress having a host?

Reply:
If we configure ingress with a hostname, we can leverage https://github.com/jetstack/cert-manager to automatically issue and renew valid SSL certificates for this ingress.

Reply:
Updated the PR description as well.

Reply:
thanks